src/python-setup/python-exe.bat

```diff
@@ -1,26 +1,23 @@
 @echo off
 
-Set version=3.4.3
-Set python32Bit=python-%version%.msi
-Set python64Bit=python-%version%.amd64.msi
-Set pythonPath=%cd:~0,2%/Python34
+Set folderName=Python36
+Set version=3.6.8
+Set python32Bit=python-%version%.exe
+Set python64Bit=python-%version%-amd64.exe
+Set pythonPath=%cd:~0,2%/Python3.6.8
 Set pythonScriptPath="%pythonPath%/Scripts"
 Set pythonExe="%pythonPath%/python.exe"
 Set pipWhl=pip-6.0.8-py2.py3-none-any.whl
 
-rem Add the pythonPath to environment variables.
-echo %pythonScriptPath%
-Setx Path "%pythonScriptPath%;%Path%"
-
 rem Execute python based on machine architecture.
 If /i "%processor_architecture%"=="x86" (
   If NOT DEFINED PROCESSOR_ARCHITEW6432 (
-      msiexec /i "%python32Bit%" /passive
+      start "" "%python32Bit%" /passive DefaultJustForMeTargetDir="C:\%folderName%"
   ) Else (
-      msiexec /i "%python64Bit%" /passive
+      start "" "%python64Bit%" /passive DefaultJustForMeTargetDir="C:\%folderName%"

```

Comment:
You can also use `TargetDir` instead of `DefaultJustForMeTargetDir`.

Reply:
The `TargetDir ` will install the Python for all user.  I think lets just use the `DefaultJustForMeTargetDir ` Ref: https://docs.python.org/3/using/windows.html